sufia.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'select2-rails', '~> 3.5.9'
   spec.add_dependency 'json-schema'
   spec.add_dependency 'oauth'
+  spec.add_dependency 'devise', '~> 4.2'
```

Comment:
Fair point.

More broadly... ScholarSphere's `sufia7` branch is using devise 3.5.10. I don't see why fixing a deprecation warning in Sufia's test suite should force every Sufia adopter to update their devise dependency to `~> 4.2`.